docs/proposals/Avatar/README.md

```diff
@@ -0,0 +1,78 @@
+# Avatar
+
+An `<Avatar />` component will be used to display an identifier (image or
+initials) for a user.
+
+Rebecca Li has done some
+[great research](https://jobber.atlassian.net/browse/JOB-19028) on the usage of
+avatars in Jobber, as well as the potential usage of colors within avatars.
+
+## Interface
+
+```
+<Avatar
+    image="ImageUrl"
```

Comment:
Part of me feels like we should be specific with this and call the prop `imageURL` to be clear. Thoughts?

Reply:
Or just `url` could probably be fine? :)

Reply:
Does `url` make it feel like it can be clicked? I feel `imageUrl` is more clear then just `url`

Reply:
No strong feelings. I think `href` would make it feel clickable to me. `url` just feels simpler. But, happy whichever.

Reply:
`imageUrl` feels more intuitive to me, for what it's worth